docs/src/pages/[platform]/theming/theme-provider/react.mdx

```diff
@@ -0,0 +1,89 @@
+import { Flex } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
+import { BasicExample, CustomThemeExample } from './examples';
+
+## Usage
+
+Import the `ThemeProvider` and wrap your application with it:
+
+```jsx
+import { ThemeProvider } from '@aws-amplify/ui-react';
+
+export const App = () => (
+  <ThemeProvider>
+    <YourApplication />
+  </ThemeProvider>
+);
+```
+
+After wrapping your application in the `ThemeProvider`, you have access to theme values in your components. To use the [default theme](./default-theme) to style the components in your app, you can either:
+1. Get the theme `tokens` through the `useTheme` hook (e.g., `tokens.colors.blue[80]`)
+2. Reference the theme `tokens` directly in style props (e.g., `"purple.80"`)
```

Comment:
@dbanksdesign Lmk if you'd like to wordsmith any of this

Reply:
```suggestion
After wrapping your application in the `ThemeProvider`, you have access to all theme values in your components. To style the components in your app, you can either:
1. Get the theme `tokens` through the `useTheme` hook (e.g., `tokens.colors.blue[80]`)
2. Reference the theme `tokens` directly in style props (e.g., `"purple.80"`)
```